src/Orleans.Core.Abstractions/Runtime/GrainReference.cs

```diff
@@ -388,6 +388,23 @@ public string ToKeyString()
             }
             return String.Format("{0}={1}", GRAIN_REFERENCE_STR, GrainId.ToParsableString());
         }
+
+        public GrainReferenceKeyInfo ToKeyInfo()
```

Comment:
While self-explaining,  I wonder if public methods should have comments on them. Someone else to decide. :)

Reply:
They should, yes. We may have build warnings for public, undocumented members